docs/src/main/sphinx/admin/properties-optimizer.rst

```diff
@@ -107,3 +107,12 @@ the maximum number of joins that can be reordered at once.
 
     The number of possible join orders scales factorially with the number of
     relations, so increasing this value can cause serious performance issues.
+
+``optimizer.optimize-duplicate-insensitive-joins``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Reduces number of rows produced by joins when optimizer detects that duplicated
+join output rows can be skipped.
```

Comment:
Is there any situation where user would want to disable the flag? 
Otherwise the documentation piece is not really useful for end user, is it?
I'd omit it and but i know @mosabua would have a different opinion.

Reply:
Yeah. I'm also not sure we should document kill switches.

Reply:
I think we should _if_ we can advise users when to use it. Like "disable this when you see queries failing with xxx".
This is not the case here though.

Reply:
Well, nothing that we do should cause failures, so... :)